dependencies.yaml

```diff
@@ -428,6 +435,34 @@ dependencies:
               - *tensordict
           - {matrix: null, packages: [*pytorch_pip, *tensordict]}
 
+  # Will remove this after snap-stanford/ogb#497 is resolved.
+  # Temporarily sets the max pytorch version to 2.5 for compatibility
+  # with ogb.
+  depends_on_ogb:
+    common:
+      - output_types: [conda]
+        packages:
+          - pytorch>=2.3,<2.6a0
```

Comment:
I'm a bit confused, do we also need an upper bound pinning for pip packages? It appears to be reusing `*pytorch_pip` below, which won't have the `<2.6a0` upper bound.